pwa-shell/src/ui/menu/SearchBar.js

```diff
@@ -14,10 +14,22 @@ import { FormattedMessage } from 'react-intl.macro';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 class SearchBar extends Component {
+  state = {
+    searchTerm: '',
```

Comment:
why it's in both local and global (redux) state?

Reply:
this was because when using Input's `value` property it should be paired with `onChange` but I didn't want to affect the prop when value has been changed. But I just found out that you can use `defaultValue` to achieve the same result, so I removed the local state anyways. Thank you for pointing this.